client/src/App.tsx

```diff
@@ -82,7 +82,7 @@ export default function App() {
     return (
 
         <Routes>
-            <Route path="/" element={<Home login={login} logout={logout} loginErrorMessage={errorMessage} error={error} setError={setError} user={user} />} />
+            <Route path="/" element={<Home user={user} />} />
```

Comment:
<img width="1440" alt="Screen Shot 2024-12-13 at 18 39 57" src="https://github.com/user-attachments/assets/79e1cce4-f85a-4f51-8cb8-6158b2369b05" />
It seems that it's not working on this screen. I also checked on the other sizes. it's the same

Reply:
i m witing for what to write on the carousel, i thnk we are going to change those description and images